### DISCUSSION TITLE
why it can not install v on ubuntu

### DISCUSSION THREAD

**Discussion by program-spiritual** [vlang/v#19393] [Installation Issues]
```
root@xxx:~/github/v# make
cd ./vc && git clean -xf && git pull --quiet
cd ./thirdparty/tcc && git clean -xf && git pull --quiet
cc  -std=gnu99 -w -o v1.exe ./vc/v.c -lm -lpthread
cc: fatal error: Killed signal terminated program cc1
compilation terminated.
make: *** [GNUmakefile:113: all] Error 1
```
it should not build gnu99

**Comment by JalonSolov** [comment]
Which release of ubuntu?  V is built all the time on ubuntu, in the CI, on every PR.

**Reply by program-spiritual** [reply]
ubuntu server 22.04 . 
the step is followed by v docs
clone the repo 
build by make command

**Comment by jacksonmowry** [comment]
Do you have gcc installed? I recently deployed to an Ubuntu VPS and was surprised to see that they don't ship gcc with the latest version.

**Reply by program-spiritual** [reply]
i use the `apt install gcc ` to install it , hovever ,the same  result as it is
